src/Calculator/Controls/OverflowTextBlock.cpp

```diff
@@ -30,25 +30,38 @@ DEPENDENCY_PROPERTY_INITIALIZATION(OverflowTextBlock, TokensUpdated);
 
 void OverflowTextBlock::OnApplyTemplate()
 {
-    assert(((m_scrollLeft == nullptr) && (m_scrollRight == nullptr)) || ((m_scrollLeft != nullptr) && (m_scrollRight != nullptr)));
+    UnregisterEventHandlers();
```

Comment:
slight refactoring to make this control more customizable (in case developers want to use it in their own apps):
- can apply a template than 1 time
- make controls optional